internal/services/mssql/mssql_server_resource_test.go

```diff
@@ -71,28 +86,28 @@ func TestAccMsSqlServer_update(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep("administrator_login_password"),
+		data.ImportStep("administrator_login_password", "minimum_tls_version"),
```

Comment:
this is indicative of a bug in the code. How come this is required here?

Reply:
@katbyte thanks for your comment. There is no API bug here. I have removed `minimum_tls_version`.